spec.bs

```diff
@@ -868,12 +902,16 @@ A <dfn export for=fencedframetype>destination event</dfn> is either a
     : [=request/credentials mode=]
     :: `"omit"`
 
-  1. [=Fetch=] |request|.
+    : [=request/Attribution Reporting eligibility=]
+    :: |attributionReportingEligibility|
+
+  1. [=Fetch=] |request| with [=fetch/processResponse=] being |processResponse| if it is not null
+     and [=fetch/useParallelQueue=] being |useParallelQueue|.
 
     Note: This algorithm can be invoked from while [=in parallel=] or while on a {{Document}}'s main
-    thread. To handle the [=in parallel=] invocation correctly, ordinarily we would invoke [=fetch=]
-    with [=fetch/useParallelQueue=] set to true, but since we don't pass in any of the <span
-    class=allow-2119>optional</span> [=response=]-handling algorithms, there is no need to do this.
+    thread. To handle the [=in parallel=] invocation correctly, we invoke [=fetch=]
+    with [=fetch/useParallelQueue=] set to true when the <span class=allow-2119>optional</span>
+    [=response=]-handling algorithm is used, otherwise there is no need to do this.
```

Comment:
Can you link to any web platform tests here that rely on the response headers being processed by https://wicg.github.io/attribution-reporting-api/#process-an-attribution-eligible-response?

Reply:
https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/web_tests/external/wpt/attribution-reporting/
https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/web_tests/wpt_internal/attribution-reporting/
Most of the tests in these two suites rely on that.

Reply:
Great, can you link to some of them via a block like this: https://github.com/WICG/fenced-frame/blob/master/spec.bs#L1660-L1666 ? Just the external/wpt ones will be linkable with that.

Reply:
Actually those wpts are for attribution reporting only, not fenced frame related, so I think it's better to link to ARA spec.

Reply:
OK, that sounds fine.